bundle/compliance/cis_gcp/rules/cis_5_1/rule.rego

```diff
@@ -0,0 +1,17 @@
+package compliance.cis_gcp.rules.cis_5_1
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import data.compliance.policy.gcp.iam.ensure_no_public_access as audit
```

Comment:
Rules 1.9, 5.1, 7.1 are all similar and refer to the iam policy of the resource, therefore, I thought it makes sense to keep it under iam but it can be a global audit as well.
wdyt?